Sources/BraintreeCore/BTGraphQLErrorTree.swift

```diff
@@ -20,7 +20,7 @@ class BTGraphQLErrorTree {
             if !parentNode.hasChild(forField: field) {
                 parentNode.insertChild(BTGraphQLMultiErrorNode(field: field))
             }
-            parentNode = parentNode.getChild(forField: field) as! BTGraphQLMultiErrorNode
+            parentNode = parentNode.getChild(forField: field) as? BTGraphQLMultiErrorNode ?? BTGraphQLMultiErrorNode(field: "Failed to find BTGraphQLMultiErrorNode for field: \(field)")
```

Comment:
I wonder if Lines 20-23 could be reduced to something like
```
parentNode = parentNode.createChildIfNotExists(forField: field)
```
The name is total garbage, but it'll avoid having to force unwrap. Technically the right side of the null coalesce won't ever run as it's written.